### PR TITLE
[#958911] Add some media wrapper docs and deprecate the player docs, and...

### PR DESCRIPTION
**Pull request by ScottDowne**
... some other small fixes.

### CODE REVIEW

File: media-wrappers/_posts/2014-03-24-null.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+The null wrapper has numerous use cases, such as :
+
+* Wanting to use popcorn without a media
+* Creating a bigger global timeline in which numerous smaller popcorn instances are created from
+
+## Examples ##
+
+{% highlight js linenos %}
+    var wrapper = Popcorn.HTMLNullVideoElement( "#elementID" );
```
Comment (mjschranz) [inline media-wrappers/_posts/2014-03-24-null.md:24]: Wouldn't we technically still need to give a `src` after for this to work?

File: media-wrappers/_posts/2014-03-24-vimeo.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+* **id** \[String\] - the id of the HTML element that the youtube wrapper will populate
+
+* Note - Things like width and height are inherited from the HTML element that the wrapper is populating, and if are not present, default to a width of 350px and a height of 300px
+
+## Examples ##
+
+{% highlight js linenos %}
+    var wrapper = Popcorn.HTMLVimeoVideoElement( "#video" );
+
+    wrapper.src = "vimeoUrl";
```
Comment (mjschranz) [inline media-wrappers/_posts/2014-03-24-vimeo.md:21]: Going to use an actual Vimeo source just for completeness sake?

File: media-wrappers/index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: Media Wrappers
+---
+<nav class="main">
+  <h1><a href="/popcorn-docs/index.html">API Reference</a></h1>
+
+  <ul>
+    <li><a href="/popcorn-docs/getting-started">Getting Started</a></li>
+		<li><a href="/popcorn-docs/addon-development">Addon Development</a></li>
```
Comment (mjschranz) [inline media-wrappers/index.html]: Line up with other elements.

File: index.html [hunk 4]
```diff
@@ -15,6 +15,7 @@
 		    <li><a href="/popcorn-docs/utility-methods">Utility Methods</a></li>
 		    <li><a href="/popcorn-docs/events">Events</a></li>
 		    <li><a href="/popcorn-docs/plugins">Plugins</a></li>
+        <li><a href="/popcorn-docs/media-wrappers">Media Wrappers</a></li>
```
Comment (mjschranz) [inline index.html]: Line up with other LIs

File: media-wrappers/_posts/2014-03-24-vimeo.md (hunk 2 above)
Comment (ScottDowne) [inline media-wrappers/_posts/2014-03-24-vimeo.md:21]: I'm not a big fan of using random videos in examples like this. They tend to rot or age badly.

However, all I did was copy and paste the previous player examples and update what was needed.

I don't think we should worry about the source.

**Pull request merged**